README.md

```diff
@@ -112,8 +112,11 @@ their combination.
 
 We also have explicit _non-goals_ for Carbon, notably including:
 
--   A stable ABI for the entire language and library
+-   A stable [application binary interface](https://en.wikipedia.org/wiki/Application_binary_interface) (ABI) for the entire language and library
```

Comment:
This is definitely a good change.

Could you separate this and the other changes into two PRs so that we can go ahead and land this one?

Reply:
Will do.

Reply:
This PR affects the above change only. The expansion of the non-goal section is to be found in https://github.com/carbon-language/carbon-lang/pull/1698.